lib/synth-audio.js

```diff
@@ -591,11 +603,35 @@ const synthElevenlabs = async(logger, {credentials, stats, language, voice, text
     });
     return mp3;
   } catch (err) {
-    logger.info({err}, 'synthEvenlabs returned error');
+    logger.info({err}, 'synth Elevenlabs returned error');
+    stats.increment('tts.count', ['vendor:elevenlabs', 'accepted:no']);
     throw err;
   }
 };
 
+const synthWhisper = async(logger, {credentials, stats, voice, text}) => {
+  const {api_key, model_id, baseURL, timeout} = credentials;
+  try {
+    const openai = new OpenAI.OpenAI({
+      apiKey: api_key,
+      timeout: timeout || 5000,
+      ...(baseURL && {baseURL})
+    });
+
+    const mp3 = await openai.audio.speech.create({
```

Comment:
can you log how long it takes to generate the audio ?  That would be useful

Reply:
This log.info is already done the job:
https://github.com/jambonz/speech-utils/pull/41/files#diff-b2da89e3647bf37b28429f697620c5a02e7e41387bd799d60eb1f8577330f9f7R214